knowledge-base/grid-column-in-custom-component-is-last.md

```diff
@@ -34,7 +34,7 @@ This behavior stems from the order in which the framework initializes components
 
 All parent components are initialized before their child components.
 
-On initialization, child components can't be rendered before the parent components. This is why the order in the actual code can't affect the order on initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized.
+On initialization, child components can't be rendered before the parent components. This is why the order in the actual code can't affect the order on initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized. The columns that are added later in the component lifecycle will also be last.
```

Comment:
Should be "order **of** initialization":

```suggestion
On initialization, child components can't be rendered before the parent components. This is why the order in the actual code can't affect the order of initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized. The columns that are added later in the component lifecycle will also be last.
```

Also, the following statement may raise the question "What determines the order of column initialization, if it isn't the order in the code?":

"the actual code can't affect the order of initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized"

Finally, what does "added later in the component lifecycle` mean? Is it programmatic addition of columns or just "later" in the code?